spring-ai-core/src/main/java/org/springframework/ai/observation/conventions/VectorStoreProvider.java

```diff
@@ -23,6 +23,22 @@ public enum VectorStoreProvider {
 
 	// @formatter:off
         PG_VECTOR("pg_vector"),
+        AZURE_VECTOR_STORE("azure_vector_store"),
```

Comment:
I would keep these names framework-agnostic for better interoperability with LLM observability backends. For example, I would use `chroma` and `elasticsearch` instead of `chroma_vector_store` and `elasticsearch_vector_store`

Reply:
Agree. It also improves the readability.